src/main/resources/application.yaml

```diff
@@ -125,3 +125,7 @@ idam:
     authorization: ${OAUTH2_AUTH:}
     client-authorization: ${OAUTH2_CLIENT_AUTH:}
 
+
+elinksApiKey: ${judicial_api_elinks_api_key:}
+#URL not tested , code is for developers reference
+elinksUrl: ${ELINKS_URL:https://jo-staging.elinks.judiciary.uk}
```

Comment:
The elink url should be configured at the environment specific, in all the lower environments its same staging api. in production different url

Reply:
did changes in values.yaml , for prod environment changes pushed in cnp-flux